src/components/date-picker/DatePicker.tsx

```diff
@@ -150,6 +152,8 @@ export interface DatePickerProps extends WrappedComponentProps {
     placeholder?: string;
     /** Resin tag */
     resinTarget?: string;
+    /** Does the date input meet accessibility standards */
```

Comment:
Can we add a `@deprecated` to the displayFormat field mentioning it won't be supported with the accessible mode turned on?